src/app/modules/item/services/item-datasource.service.ts

```diff
@@ -34,8 +34,8 @@ export class ItemDataSource implements OnDestroy {
   private readonly refresh$ = new Subject<void>();
   private readonly scorePatch$ = new Subject<number | undefined>();
   private readonly maxScorePatch$ = this.scorePatch$.pipe(
-    // Keep max score of all emitted scores **for current item**
-    // maxScorePatch is resetted to undefined at each refresh (see subscriptions below)
+    // Keep max score of all emitted scores. NB: adding operators to a subject makes it COLD.
+    // Since it is cold, mean of max scores is ONLY computed for values emitted during the lifetime of the subscription
```

Comment:
why "mean"?

Reply:
:thinking: indeed, it's max. I'll change it